light_curves/requirements_light_curve_generator.txt

```diff
@@ -14,7 +14,7 @@ astroquery>=0.4.8.dev0
 acstools
 lightkurve
 alerce
-lsdb
+lsdb==0.4.5
```

Comment:
is this a minimum requirement or strictly only this version works?

Reply:
How do I test this?  It doesn't work without the  ==0.4.5 which is the current version.  Is it best practice to make this a >= and assume future versions will work?

Reply:
yeap, add `>=` as we go with the assumption future versions work, too. If not, then we introduce an upper pin or change our code.